src/modules/integrator-guide/pages/automation/workflows/examples/template-rendering.adoc

```diff
@@ -0,0 +1,164 @@
+= Template rendering
+:attachment-path: ../../_attachments/workflow/
+
+In this example, we'll take a look at how to render different templates that were made in our xref:templates/index.adoc[template system].
+
+To start, create a template of type HTML in our templating system located in the admin area.
+
+== HTML
+
+Create a workflow similar to the one below. Make sure to look at the step details of steps (2) and (3).
+
+.The screenshot outlines the workflow that renders an HTML template and sends it as an email.
+image::workflow/example-template-rendering/html-workflow.png[role="data-zoomable"]
+
+The link:{attachment-path}template_rendering_html.json[source code] for the workflow example.
+
+
+****
+.Workflow step details:
+[%collapsible.result]
+====
+1. *(1) System; onManual*:
+** *resource*: `System`
+** *event*: `onManual`
+** *enabled*: checked
+2. *(2) Render template*:
+** *type*: `Template render`
+** *arguments*:
+*** *lookup*:
+**** *type*: `Handle`
+**** *value type*: constant
+**** *value*: `email-template`
+*** *documentName*:
+**** *value type*: constant
+**** *value*: `Email template`
+*** *documentType*:
+**** *value type*: constant
+**** *value*: `text/html`
+** *results*:
+*** *document target*: `renderedTemplate`
+3. *(3) Send email*:
+** *type*: `Email`
+** *arguments*:
+*** *subject*:
+**** *type*: `String`
+**** *value type*: constant
+**** *value*: `Email template`
+*** *from*:
+**** *type*: `String`
+**** *value type*: constant
+**** *value*: `demo@mail.com`
+*** *to*:
+**** *type*: `String`
+**** *value type*: constant
+**** *value*: `test@mail.com`
+*** *html*:
+**** *type*: `Reader`
+**** *value type*: expression
+**** *value*: `renderedTemplate.document`
+** *results*:
+*** *document target*: `content`
+====
+****
+
+[CAUTION]
+====
+Make sure that the `documentType` parameter of the `(2) Render template` is of value `text/html`
+====
+
+== PDF
+The difference between HTML and PDF rendering is the `documentType` parameter changes to `application/pdf`. With PDFs you can also adjust the xref:integrator-guide:templates/index.adoc#tpl-render-options_marginBottom[render options] to modify how the final PDF is rendered.
```

Comment:
One sentence per line.


`...between HTML and PDF rendering is the `documentType` parameter changes to...` -> `...between HTML and PDF rendering is that the `documentType` parameter changes to...`